content/en/real_user_monitoring/browser/troubleshooting.md

```diff
@@ -100,6 +100,13 @@ If an event or a request goes beyond any of the following limitations, it is rej
 | Maximum event size                       | 256 KB       |
 | Maximum intake payload size              | 5 MB         |
 
+## Cross origin read blocking warning
+
+When starting session replay recording with `startSessionReplayRecording()` on Chromium based browser, a CORB warning is printed in the console:
+{{< img src="real_user_monitoring/browser/troubleshooting/corb_warning.png" alt="CORB warning">}}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I would write the error message in full text so it is searchable

Reply:
Right, good idea